lib/ImportFromC/HeaderToModel.cpp

```diff
@@ -26,6 +26,7 @@ static constexpr llvm::StringRef RawABIPrefix = "raw_";
 
 static constexpr llvm::StringRef ABIAnnotation = "abi:";
 static constexpr llvm::StringRef RegAnnotation = "reg:";
+static constexpr llvm::StringRef StackAnnotation = "stack";
```

Comment:
Can you separate the rework of `stack` in a seprarate commit than the rework of parsing?

Reply:
I looked into it, but that's going to require non-trivial effort. Since the "pre-rework" version was pretty adamant about having single annotation per argument - and then dealing with how to parse it. The new version treats each annotation as its own thing - hence the need to differentiate the two.

Reply:
Ok :shrug: